plugins/wasi_nn/ggml.cpp

```diff
@@ -81,16 +78,6 @@ Expect<ErrNo> parseMetadata(Graph &GraphRef, const std::string &Metadata,
     GraphRef.ReversePrompt = ReversePrompt;
   }
 
-  // The model parameters.
-  if (Doc.at_key("n-gpu-layers").error() == simdjson::SUCCESS) {
```

Comment:
Should we keep this segment of code to make it still be able to configure ngl using metadata?

Reply:
Fixed.